src/bindings/pyrpr/src/pyrprapi.py

```diff
@@ -663,7 +663,8 @@ def get_rif_sdk(base=Path()):
                  'RPR_SHAPE_SET_LIGHTMAP_CHART_INDEX_FUNC_NAME',
                  'RPR_MESH_UPDATE_FUNC_NAME',
                  'RPR_SHAPE_SET_TRANSFORM_BATCH_FUNC_NAME',
-                 'rprDirectionalLightSetRasterShadowSplits']
+                 'rprDirectionalLightSetRasterShadowSplits',
+                 'RPR_CONTEXT_CREATE_MESH_EDITABLE_FUNC_NAME',]
```

Comment:
Currious.  What does this function do?

Reply:
It's from the RadeonProRender_Baikal.h headers file. Other entries in that list are from the same header as well.